pifpaf/drivers/__init__.py

```diff
@@ -25,6 +25,7 @@
 import fixtures
 import jinja2
 import six
+import xattr
```

Comment:
xattr is not available on Windows so it's a bit problematic to import it without a condition.

That was the main reason of having the `ceph` flavor.